youtube.module

```diff
@@ -598,19 +598,34 @@ function youtube_field_widget_form(&$form, &$form_state, $field, $instance, $lan
       '#suffix' => '</div>',
       '#weight' => 1,
     );
-  }
 
+  }
   if (isset($start_time) && $start_time > 0) {
     $element['video_start_time'] = array(
       '#prefix' => '<div class="youtube-video-start-time">',
       '#markup' => t('YouTube video start time: !video_start_time', array(
-        '!video_start_time' => date("H:i:s", $start_time),
+        '!video_start_time' => gmdate("H:i:s", $start_time),
```

Comment:
This method has the limitation that it can't convert numbers bigger than 86,400 (24 hours). See https://stackoverflow.com/questions/3172332/convert-seconds-to-hourminutesecond for alternatives.

Reply:
Thanks for the tip. Unlikely to be an issue with YouTube videos, but done anyway in case the limits change.